pkg/cli/settings.go

```diff
@@ -14,30 +14,23 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-/*Package cli describes the operating environment for the Helm CLI.
-
-Helm's environment encapsulates all of the service dependencies Helm has.
-These dependencies are expressed as interfaces so that alternate implementations
-(mocks, etc.) can be easily generated.
-*/
 package cli
 
 import (
 	"fmt"
-	"os"
-	"strconv"
-
 	"github.com/spf13/pflag"
-	"k8s.io/cli-runtime/pkg/genericclioptions"
-
 	"helm.sh/helm/v3/pkg/helmpath"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"os"
+	"strconv"
 )
 
-// EnvSettings describes all of the environment settings.
-type EnvSettings struct {
```

Comment:
by changing the name, we are breaking backwards compatibility. To ensure other programs do not fail to compile after upgrading helm, Go has a feature called [type aliasing](https://golang.org/doc/go1.9#language).

```
type EnvSettings = Settings
```

This allows users to use either type, and it allows us to maintain backwards compatibility.